agent/agent.go

```diff
@@ -98,11 +98,22 @@ func (a *Agent) unloadUnit(unitName string) {
 	a.registry.ClearUnitHeartbeat(unitName)
 	a.cache.dropTargetState(unitName)
 
-	a.um.TriggerStop(unitName)
+	errStop := a.um.TriggerStop(unitName)
+	if errStop != nil {
+		log.Warningf("Failed stopping unit(%s): %v", unitName, errStop)
```

Comment:
nit - these messages are a bit misleading - because it's just the trigger, not the actual stopping, right?